tests/test_rs_basics.py

```diff
@@ -659,7 +659,13 @@ def test_write_texture2():
 
 def test_parse_shader_error(caplog):
 
-    # invalid attribute access
+    device = wgpu.utils.get_default_device()
+
```

Comment:
I would recommend to use the following function to determine if the terminal you are outputting to supports colors. Many terminals don't!

```python
def terminal_supports_colors():
    term_isatty = sys.stdout.isatty()
    term_smart = os.environ.get("TERM", "") not in {"", "dumb", "unknown"}
    return term_isatty and term_smart
```

Reply:
Learned something new. :)
I will update it after a little while.